client/src/components/AllPatients/AllPatients.jsx

```diff
@@ -59,12 +59,15 @@ function AllPatients({
               </thead>
               <tbody>
                 {patients.map((patient) => (
-                  <tr key = {patient.id}>
+                  <tr key={patient.id}>
                     <td>
                       <Link
-                        to={`/viewpatient/${patient.id}/${capitalizeName(patient.firstname)}/${capitalizeName(patient.lastname)}`}
+                        to={`/viewpatient/${patient.id}/${capitalizeName(
+                          patient.firstname
+                        )}/${capitalizeName(patient.lastname)}`}
                       >
-                        {capitalizeName(patient.firstname)} {capitalizeName(patient.lastname)}
+                        {capitalizeName(patient.firstname)}{" "}
+                        {capitalizeName(patient.lastname)}
```

Comment:
I'm curious if this works? 
`{capitalizeName(patient.firstname) . " " . capitalizeName(patient.lastname)}`

There should be a way to do string concatenation in typescript if you want to look it up.